test/indexer_unit_test.go

```diff
@@ -55,6 +55,24 @@ func IndexerUnitTestContext(s *godog.Suite) {
 	s.Step(`^we make a Search Accounts call with exclude "([^"]*)"$`, weMakeASearchAccountsCallWithExclude)
 	s.Step(`^we make a Lookup Account by ID call against account "([^"]*)" with exclude "([^"]*)"$`, weMakeALookupAccountByIDCallAgainstAccountWithExclude)
 	s.Step(`^we make a SearchForApplications call with creator "([^"]*)"$`, weMakeASearchForApplicationsCallWithCreator)
+	s.Step(`^we make a LookupApplicationBoxByIDandName call with applicationID (\d+) with encoded box name "([^"]*)"$`,
+		func(appId int, encodedBoxName string) error {
+			indexerClient, err := indexer.MakeClient(mockServer.URL, "")
+			if err != nil {
+				return err
+			}
+			indexerClient.LookupApplicationBoxByIDandName(uint64(appId)).Name(encodedBoxName).Do(context.Background())
+			return nil
+		})
+	s.Step(`^we make a SearchForApplicationBoxes call with applicationID (\d+) with max (\d+)$`,
```

Comment:
I think this should also include a `next "([^"]*)"$`. Also, I'm not familiar enough with this repo to understand if the same `next` encoding problem I [hilighted in the python SDK](https://github.com/algorand/py-algorand-sdk/pull/366/files#r948447135), applies here.

Reply:
good point, just add the segment for `next`.